src/frontend/sharedComponents/MediaScrollView/AudioThumbnail.tsx

```diff
@@ -65,6 +65,8 @@ export const AudioThumbnail: FC<AudioThumbnailProps> = ({
 
   return (
     <TouchableOpacity
+      accessible={true}
```

Comment:
Minor, but not necessary to explicitly specify this since all touchables are marked as accessible by default

https://reactnative.dev/docs/0.76/accessibility#accessible

> By default, all touchable elements are accessible.

Doesn't hurt to leave it but just wanted to point it out.